sycl/source/detail/devicelib/CMakeLists.txt

```diff
@@ -45,8 +86,52 @@ add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-cassert.spv
                    DEPENDS wrapper.h clang llvm-spirv
                    VERBATIM)
 
-add_custom_target(devicelib-obj DEPENDS ${devicelib-obj-file})
-add_custom_target(devicelib-spv DEPENDS ${binary_dir}/libsycl-fallback-cassert.spv)
+add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-complex.spv
+                   COMMAND ${clang} -S -fsycl-device-only -fno-sycl-use-bitcode
```

Comment:
I think we should remove `-S` here and from all custom commands below.
If I understand it correctly, today compiler dosn't support it, but it implies textual format of SPIR-V file whereas this command must produce binary format.